CHANGELOG.md

```diff
@@ -128,6 +128,14 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Style any custom HTML in your cookie banner
+
+We have removed the default font styles from the cookie banner Sass, to make it more obvious when classes/styles haven't been added to any custom HTML.
```

Comment:
```suggestion
We've removed the default font styles from the cookie banner Sass. This change makes it more obvious when classes and styles have not been added to any custom HTML.
```
Style guide encourages us to use positive contractions, but to avoid negative contractions (some users find them hard to read). Also says to replace '/' with 'or'.

Reply:
Thanks @EoinShaughnessy , really useful to know! I'll try to remember for future 😬